src/ActionModal/components/DelegationModal.vue

```diff
@@ -52,7 +58,37 @@
     </TmFormGroup>
 
     <TmFormGroup
-      v-if="fromOptions.length > 1"
+      v-else
+      class="action-modal-form-group"
+      field-id="to"
+      field-label="To"
+    >
+      <TmField
+        id="to"
+        v-model="targetValidator.operatorAddress"
```

Comment:
how much work would it be to show the picture (like we do in the activity page) in the dropdown 😄 ?

Reply:
and, would it be possible (how much work?) if we changed the page of the validator when a user selected a validator from this list? would that be strange ux? just a thought...

Reply:
#scopecreep

Reply:
> and, would it be possible (how much work?) if we changed the page of the validator when a user selected a validator from this list? would that be strange ux? just a thought...

mmm I think would be strange :-), but useful because then users could see helpful information about the possible validators to redelegate (like comission or voting power), half hided by the modal window thought xD.

mmm may be a dumb question but why we need modal windows? Shouldn't be easier to use the same layaout for all actions? For example in this case we are limited by the modal window size...

Just a thought! don't want to reinvent the world ;-)

Reply:
> #scopecreep

**Definitely, but sometimes it's nice to take these opportunities to discuss new ideas - however always good to assume these things will happen in another PR! 

> why we need modal windows

what do you mean @mariopino ?